.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
```

Comment:
I suddenly feel sad. (but OCaml is not listed [here](https://docs.travis-ci.com/user/languages/))

Reply:
indeed :(

Reply:
issue about it here: https://github.com/ocaml/ocaml-ci-scripts/issues/53 ; there is a path for becoming a "community-supported language", which requires at least 3 community contributors interested in maintaining the support.  We have 2 so far!